utils/python/CIME/XML/grids.py

```diff
@@ -150,9 +150,9 @@ def _get_gridmaps(self, component_grids, atmnlev=None, lndnlev=None):
                             logger.debug(" %s: %s" %gridmap)
 
         if atmnlev is not None:
-            grids["atm_grid"] += "z"+atmnlev
+            grids[0] += "z"+atmnlev
```

Comment:
This is the change that confused me in this file. I'm OK with this change, but the fact that we are using lists of tuples in this file at all seems to be a weird Perl-ism. I've created a separate issue for that, #232.